src/components/DeckMechanisms/play.js

```diff
@@ -9,45 +10,40 @@ export const DEFAULT_PLAY_OPTIONS = {
 /**
  * Mutate the given state following a play.
  * @param {Object} state - State being mutated
- * @param {String} id - Id of the played card
+ * @param {Object} card - Object containing id and idx of the played card
```

Comment:
Maybe we could do something like:
```suggestion
 * @param {DRCard} card - Played card
```
… as a hint for us that this means `{ id, idx }`. I guess this is one of these cases where TypeScript would be handy, but heh.